plugins/package-managers/maven/src/funTest/assets/projects/synthetic/maven-expected-output-app.yml

```diff
@@ -3,8 +3,8 @@ project:
   id: "Maven:org.ossreviewtoolkit.maven:maven-app:1.0-SNAPSHOT"
   definition_file_path: "plugins/package-managers/maven/src/funTest/assets/projects/synthetic/maven/app/pom.xml"
   authors:
-  - "Google"
-  - "The Apache Software Foundation"
+  - "ACME"
```

Comment:
Commit message: Please don't only write *what* you're doing, but also *why*.

Reply:
The actual reason is that @mnonnenmacher asked me to do so, but this would be a strange commit message. I have reworded the message.